class.wp-cldr.php

```diff
@@ -146,6 +153,7 @@ public function get_cldr_locale( $wp_locale ) {
 		);
 
 		// Convert underscores to dashes and everything to lowercase.
+		$cleaned_up_wp_locale = '';
```

Comment:
This line will never affect the function output as the variable is reassigned in the next line without regard for this one.